app/models/WorkItem.cs

```diff
@@ -1,15 +1,14 @@
+using System.Security.Cryptography;
```

Comment:
This looks like it's maybe a dependency that we should remove. NBD but we can tackle it later :)